Rocket-Nozzle-Design-and-Analysis/Rocket_Nozzle_Design_and_Analysis.tex

```diff
@@ -0,0 +1,235 @@
+
+\section{Primary Objective}\label{sec:primary-obj}
+The primary goal of this project is to use the analytical rocket equations to design a conical rocket nozzle to produce 100 Newtons of thrust.
+The team will design a new stand that is capable of securely fastening a propulsion test of 200 Newtons.
+Thus providing a factor of safety of 2.0. The tests will be performed with Air or Nitrogen so the extensive tabulated data can be used in the performance analysis of the nozzle.
+
+\section{Benefit to SPEX}\label{sec:benefit}
+The benefit to SPEX would be the further education in the design of rocket nozzles using the analytical rocket equations. The experiment platform would
+be a great exhibit for visiting companies or Imagine RIT where SPEX would be on display. This project would be
+the first time that the propulsion team has fully designed and operated a rocket nozzle. Using these equations correctly is very complicated and important for
+propulsion engineers to understand. The project would have a full life cycle which includes design, manufacture, test, and reporting the results.
+
+\section{Implementation}\label{sec:implementation}
+The project will start with a brief eduation of the rocket equations. The team will go through each one so everyone on the team is acquainted with them before we dive into the
+nozzle design. Then the team will start with the nozzle design. Once the nozzle design is complete the team will design the test stand that can adequately contain the
+propulsion system. Then the team will manufacture the nozzle and test stand. After testing, if time permits, the team will use a schlieren lensing setup to
+get a very good look at the fluid flowing out of the nozzle.
+
+\subsection{Deliverables}\label{subsec:deliverables}
+The physical deliverables for this project are the test stand and rocket nozzle. The test stand and rocket nozzle can be shown to faculty and other
+bodies that could support SPEX in some fashion. The test stand and nozzle will be stored so it can be demonstrated when necessary, such as an introduction to SPEX
+engineering to students, faculty, or others.
+
+The non-physical deliverables would be a poster that exhibits the work completed and the results of the experiment. There will be a final report that outlines
+the method used for designing the nozzle and manufacturing the nozzle and test stand. The report will be written in the IEEE standard and can be submitted to various
+organizations for funding other outreach opportunities.
+
+\subsection{Milestones}\label{subsec:milestones}
+The milestones for this project are, in chronological order; nozzle design, test stand design, nozzle fabrication, test stand fabrication, testing, and final report.
+The nozzle design and fabrication proceed the test stand because the test stand must be designed to fit the nozzle, the nozzle should not be designed to fit the test stand.
+The following, is a proposed project schedule.
+
+\begin{table}
```

Comment:
Is it possible to run nozzle design concurrently with test stand design and fabrication? This would be dramatically shorten the project timeline, which would give buffer time or additional time for testing at the end of the schedule. If there is enough interest, could a dedicated subteam fabricate the test stand, given known maximum parameters? 200 Newtons is already defined earlier in the document.